### PR TITLE
Fix/try another link

### PR DESCRIPTION
**Pull request by sumitnitsurat**
- Resetting the certificate state to initial state.

### CODE REVIEW

File: src/reducers/certificate.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ export const initialState = {
 
 // Actions
 export const types = {
+  RESET_CERTIFICATE: "RESET_CERTIFICATE",
```
Comment (yehjxraymond) [inline src/reducers/certificate.js]: Possible to use `UPDATE_CERTIFICATE` action with empty certificate instead? That way, no need for new action.

Comment (sumitnitsurat) [reply src/reducers/certificate.js]: Can not use "UPDATE_CERTIFICATE" directly as it calls verify certificate calls subsequently.

**Review by rjchow — CHANGES_REQUESTED** [review]
add test

File: src/reducers/certificate.test.js [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
   getActiveTemplateTab,
-  getTemplates
+  getTemplates,
+  resetCertificateState
 } from "./certificate";
 
 describe("reducers", () => {
+  describe("Reset certificate", () => {
+    it("should reset the state to initial state", () => {
+      const action = {
```
Comment (rjchow) [inline src/reducers/certificate.test.js]: should use this instead
https://github.com/OpenCerts/opencerts-website/blob/e369fe2309bc03e4b4bdef083e3eed3a44aaa060/src/reducers/certificate.js#L303

Comment (sumitnitsurat) [reply src/reducers/certificate.test.js]: added test for action also.

Comment (rjchow) [reply src/reducers/certificate.test.js]: no need, just use the exported function directly instead of replicating the action generator here

**Review by rjchow — APPROVED** [review]

**Pull request merged**